package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "aegir lint",
     "build": "aegir build",
-    "test": "aegir test -t node -t browser",
+    "test": "aegir test -t node -t browser --timeout 60000",
```

Comment:
I think we want timeouts per tests instead of global timeouts - should gives us a better idea of what's slow. That's what we've been leaning towards at least.

Reply:
Thanks! I think setting short timeouts and then having logic for retrying until success is the way to go, then we don't have to fiddle with timeouts.

However, this change here is just made to confirm if things are really broken, or tests are just failing because of timeouts.

Reply:
Btw, "retrying until success" would look something like this: https://github.com/libp2p/js-libp2p/blob/10357a37f21a6c6d682970b10299a400259a0802/test/nodejs-bundle/tcp%2Bwebsockets%2Bwebrtc-star.js#L197-L228